cart/src/main/java/com/yas/cart/controller/CartController.java

```diff
@@ -34,4 +45,21 @@ public ResponseEntity<List<CartGetDetailVM>> listCartDetailByCustomerId(@PathVar
             return ResponseEntity.ok(cartService.getCartDetailByCustomerId(customerId));
         return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(null);
     }
+
+    @PostMapping(path = "/storefront/carts")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "201", description = "Created", content = @Content(schema = @Schema(implementation = CartGetDetailVM.class))),
+            @ApiResponse(responseCode = "400", description = "Bad request", content = @Content(schema = @Schema(implementation = ErrorVm.class))) })
+    public ResponseEntity<CartGetDetailVM> createCart(@Valid @RequestBody CartPostVM cartPostVM,
```

Comment:
Customer need to have access token to call this api

Reply:
Only customer can create cart for himself, don't need to check admin role

Reply:
Any request add to cart need login 1st (access token)